src/app/components/Testimonials.jsx

```diff
@@ -1,12 +1,32 @@
 import Link from "next/link";
```

Comment:
I recommend creating a folder Testimonials
Create index.jsx file inside and move this content inside
And create folder Ratings inside Testimonials folder and move common/TestimonialsRatings.jsx there

So it'll look like:
/components/Testimonials/index.jsx
/components/Testimonials/Ratings/index.jsx

Reply:
This way you'll cleanup your common folder and Ratings will be isolated inside Testimonials folder and that's for good as we don't use it anywhere else anyway